src/components/landscape-sidebar/geo-description-widget/geo-description-widget-component.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { Loading } from 'he-components';
-import ShareModal from 'components/share-modal';
 import { VIEW_MODE } from  'constants/google-analytics-constants';
 import styles from './geo-description-widget-styles.module.scss';
+import * as actions from 'actions/url-actions';
+import { connect } from 'react-redux';
+import FixedHeader from 'components/fixed-header'
+
+const GeoDescriptionWidget = (props) => {
+  const { data, loading, error, view, changeGlobe } = props;
+
+  const handleBackClick = () => {
```

Comment:
Building on the idea behind `container`/`component` pattern would be nice if you can extract the event handler (`handleBackClick`) outside of the `component` passing it as a `prop` from the `container`.